src/scss/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+/// Construct a logo image URL
+///
+/// @param {String} $logo-name
+/// @param {Color} $color
```

Comment:
This function looks to only take in two parameters, should we remove this comment or add this as a parameter into the function?